content/en/docs/concepts/workloads/pods/user-namespaces.md

```diff
@@ -137,20 +159,63 @@ use, see `man 7 user_namespaces`.
 
 ## Set up a node to support user namespaces
 
-It is recommended that the host's files and host's processes use UIDs/GIDs in
-the range of 0-65535.
+By default, the kubelet assumes host's files and host's processes use UIDs/GIDs
+in the range of 0-65535. This is the case in most Linux distributions with the
+default installation. The kubelet will assign UIDs/GIDs higher than that to
+pods, so there is no overlap between UID/GIDs used by host's files and processes
+and the ones used by pods.
+
+Avoiding the overlap is important to mitigate the impact of vulnerabilities such
+as [CVE-2021-25741][CVE-2021-25741], where a pod can potentially read arbitrary
+files in the host. If the UIDs/GIDs of the pod and the host don't overlap, it is
+limited what a pod would be able to do: the pod UID/GID won't match the host's
+file owner/group.
+
+The kubelet can use a custom range for user IDs and group IDs for pods. To
+configure a custom range, the node needs to have:
+
+ * A user `kubelet` in the system (you cannot use any other username here)
+ * The binary `getsubids` installed (part of [shadow-utils][shadow-utils]) and
+   in the `PATH` for the kubelet binary.
+ * A configuration of subordinate UIDs/GIDs for the `kubelet` user (see `man 5
+   subuid` and `man 5 subgid`)
```

Comment:
```suggestion
 * A configuration of subordinate UIDs/GIDs for the `kubelet` user (see `[man 5
   subuid](https://man7.org/linux/man-pages/man5/subuid.5.html)` and `[man 5 subgid](https://man7.org/linux/man-pages/man5/subgid.5.html)`)
```

Reply:
Oh, great ideas, yes! I forget the manpages are available online :D

Reply:
Fixed, thanks!